spk/tor/src/installer.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+# Package
+PACKAGE="tor"
+DNAME="Tor"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+SSS="/var/packages/${PACKAGE}/scripts/start-stop-status"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+USER="tor"
+GROUP="users"
```

Comment:
Is access to filesystem necessary for this package? If not: `nobody` would fit better :)